<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="5" ignored="0" total="5" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="55336" started-at="2018-02-09T10:03:51Z" finished-at="2018-02-09T10:04:47Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="55336" started-at="2018-02-09T10:03:51Z" finished-at="2018-02-09T10:04:47Z">
      <class name="com.ad.adinate.testscripts.NewTest">
        <test-method status="PASS" signature="beforeMethod(java.lang.String, java.lang.String)[pri:0, instance:com.ad.adinate.testscripts.NewTest@4eb7f003]" name="beforeMethod" is-config="true" duration-ms="9709" started-at="2018-02-09T15:33:52Z" finished-at="2018-02-09T15:34:02Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Firefox]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Local]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="FAIL" signature="verifyRegistration(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.ad.adinate.testscripts.NewTest@4eb7f003]" name="verifyRegistration" duration-ms="1400" started-at="2018-02-09T15:34:02Z" data-provider="testdata" finished-at="2018-02-09T15:34:03Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[FirstName1]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[LastName1]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[9876543210]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[Testdata1@ad.com]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[pwd]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[pwd]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Expected] but found [Register]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Expected] but found [Register]at org.testng.Assert.fail(Assert.java:96)at org.testng.Assert.failNotEquals(Assert.java:776)at org.testng.Assert.assertEqualsImpl(Assert.java:137)at org.testng.Assert.assertEquals(Assert.java:118)at org.testng.Assert.assertEquals(Assert.java:453)at org.testng.Assert.assertEquals(Assert.java:463)at com.ad.adinate.testscripts.NewTest.verifyRegistration(NewTest.java:56)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.lang.reflect.Method.invoke(Method.java:498)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyRegistration -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:com.ad.adinate.testscripts.NewTest@4eb7f003]" name="afterMethod" is-config="true" duration-ms="1009" started-at="2018-02-09T15:34:03Z" finished-at="2018-02-09T15:34:04Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="PASS" signature="beforeMethod(java.lang.String, java.lang.String)[pri:0, instance:com.ad.adinate.testscripts.NewTest@4eb7f003]" name="beforeMethod" is-config="true" duration-ms="8527" started-at="2018-02-09T15:34:04Z" finished-at="2018-02-09T15:34:13Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Firefox]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Local]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="FAIL" signature="verifyRegistration(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.ad.adinate.testscripts.NewTest@4eb7f003]" name="verifyRegistration" duration-ms="1341" started-at="2018-02-09T15:34:13Z" data-provider="testdata" finished-at="2018-02-09T15:34:14Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[FirstName2]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[LastName2]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[9876543210]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[Testdata2@ad.com]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[pwd]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[pwd]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Expected] but found [Register]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Expected] but found [Register]at org.testng.Assert.fail(Assert.java:96)at org.testng.Assert.failNotEquals(Assert.java:776)at org.testng.Assert.assertEqualsImpl(Assert.java:137)at org.testng.Assert.assertEquals(Assert.java:118)at org.testng.Assert.assertEquals(Assert.java:453)at org.testng.Assert.assertEquals(Assert.java:463)at com.ad.adinate.testscripts.NewTest.verifyRegistration(NewTest.java:56)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.lang.reflect.Method.invoke(Method.java:498)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyRegistration -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:com.ad.adinate.testscripts.NewTest@4eb7f003]" name="afterMethod" is-config="true" duration-ms="803" started-at="2018-02-09T15:34:14Z" finished-at="2018-02-09T15:34:15Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="PASS" signature="beforeMethod(java.lang.String, java.lang.String)[pri:0, instance:com.ad.adinate.testscripts.NewTest@4eb7f003]" name="beforeMethod" is-config="true" duration-ms="9046" started-at="2018-02-09T15:34:15Z" finished-at="2018-02-09T15:34:24Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Firefox]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Local]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="FAIL" signature="verifyRegistration(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.ad.adinate.testscripts.NewTest@4eb7f003]" name="verifyRegistration" duration-ms="1389" started-at="2018-02-09T15:34:24Z" data-provider="testdata" finished-at="2018-02-09T15:34:25Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[FirstName3]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[LastName3]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[9876543210]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[Testdata3@ad.com]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[pwd]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[pwd]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Expected] but found [Register]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Expected] but found [Register]at org.testng.Assert.fail(Assert.java:96)at org.testng.Assert.failNotEquals(Assert.java:776)at org.testng.Assert.assertEqualsImpl(Assert.java:137)at org.testng.Assert.assertEquals(Assert.java:118)at org.testng.Assert.assertEquals(Assert.java:453)at org.testng.Assert.assertEquals(Assert.java:463)at com.ad.adinate.testscripts.NewTest.verifyRegistration(NewTest.java:56)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.lang.reflect.Method.invoke(Method.java:498)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyRegistration -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:com.ad.adinate.testscripts.NewTest@4eb7f003]" name="afterMethod" is-config="true" duration-ms="798" started-at="2018-02-09T15:34:25Z" finished-at="2018-02-09T15:34:26Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="PASS" signature="beforeMethod(java.lang.String, java.lang.String)[pri:0, instance:com.ad.adinate.testscripts.NewTest@4eb7f003]" name="beforeMethod" is-config="true" duration-ms="8690" started-at="2018-02-09T15:34:26Z" finished-at="2018-02-09T15:34:35Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Firefox]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Local]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="FAIL" signature="verifyRegistration(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.ad.adinate.testscripts.NewTest@4eb7f003]" name="verifyRegistration" duration-ms="1375" started-at="2018-02-09T15:34:35Z" data-provider="testdata" finished-at="2018-02-09T15:34:36Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[FirstName4]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[LastName4]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[9876543210]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[Testdata4@ad.com]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[pwd]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[pwd]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Expected] but found [Register]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Expected] but found [Register]at org.testng.Assert.fail(Assert.java:96)at org.testng.Assert.failNotEquals(Assert.java:776)at org.testng.Assert.assertEqualsImpl(Assert.java:137)at org.testng.Assert.assertEquals(Assert.java:118)at org.testng.Assert.assertEquals(Assert.java:453)at org.testng.Assert.assertEquals(Assert.java:463)at com.ad.adinate.testscripts.NewTest.verifyRegistration(NewTest.java:56)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.lang.reflect.Method.invoke(Method.java:498)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyRegistration -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:com.ad.adinate.testscripts.NewTest@4eb7f003]" name="afterMethod" is-config="true" duration-ms="799" started-at="2018-02-09T15:34:36Z" finished-at="2018-02-09T15:34:37Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="PASS" signature="beforeMethod(java.lang.String, java.lang.String)[pri:0, instance:com.ad.adinate.testscripts.NewTest@4eb7f003]" name="beforeMethod" is-config="true" duration-ms="7491" started-at="2018-02-09T15:34:37Z" finished-at="2018-02-09T15:34:44Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Firefox]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Local]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="FAIL" signature="verifyRegistration(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.ad.adinate.testscripts.NewTest@4eb7f003]" name="verifyRegistration" duration-ms="1429" started-at="2018-02-09T15:34:44Z" data-provider="testdata" finished-at="2018-02-09T15:34:46Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[FirstName5]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[LastName5]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[9876543210]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[Testdata5@ad.com]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[pwd]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[pwd]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Expected] but found [Register]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Expected] but found [Register]at org.testng.Assert.fail(Assert.java:96)at org.testng.Assert.failNotEquals(Assert.java:776)at org.testng.Assert.assertEqualsImpl(Assert.java:137)at org.testng.Assert.assertEquals(Assert.java:118)at org.testng.Assert.assertEquals(Assert.java:453)at org.testng.Assert.assertEquals(Assert.java:463)at com.ad.adinate.testscripts.NewTest.verifyRegistration(NewTest.java:56)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.lang.reflect.Method.invoke(Method.java:498)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyRegistration -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:com.ad.adinate.testscripts.NewTest@4eb7f003]" name="afterMethod" is-config="true" duration-ms="795" started-at="2018-02-09T15:34:46Z" finished-at="2018-02-09T15:34:47Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
      </class> <!-- com.ad.adinate.testscripts.NewTest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
